bandcamp_dl/bandcamp_dl.py

```diff
@@ -33,21 +31,26 @@
     http://twitter.com/miniarray
     http://github.com/miniarray
 
+Anthony Forsberg:
+    http://evolution0.github.io
+    http://github.com/evolution0
+
 Iheanyi:
     Feel free to use this in any way you wish. I made this just for fun.
-    Shout out to darkf for writing a helper function for parsing the JavaScript! """
-
+    Shout out to darkf for writing a helper function for parsing the JavaScript!
+"""
 
-from docopt import docopt
-from Bandcamp import Bandcamp
-from BandcampDownloader import BandcampDownloader
 import os
+from docopt import docopt
+from Bandcamp_Parse import Bandcamp
+from Bandcamp_Download import BandcampDownloader
```

Comment:
Could we change it back to what they were before? I'm in agreement that it should be lowercased filenames + uppercase class names. :)